app/models/miq_request_workflow.rb

```diff
@@ -365,7 +370,6 @@ def self.parse_ws_string(text_input, options = {})
   def self.parse_request_parameter_hash(parameter_hash, options = {})
     parameter_hash.each_with_object({}) do |param, hash|
       key, value = param
-      next if value.blank?
```

Comment:
@abellotti Are you ok with this?  We have logic in automate requests that accept nil/"" for zone assignment which breaks if the keys are not passed along.